lib/middleware.js

```diff
@@ -161,6 +166,7 @@ var middlewares = module.exports = {
       'Cmd': [],
       'Dns': null,
       'Image': null,
+      'Labels': keypath.get(req, 'body.Labels') || {},
```

Comment:
below, there's a `data = assign(data, req.body);`. You can default this `'Lables'` to `{}` if you like, and let the `assign` set the value from the `body`